shell/packages/sandstorm-ui-grainlist/grainlist-client.js

```diff
@@ -357,6 +357,14 @@ Template.sandstormGrainTable.events({
     }
   },
 
+  "click td.select-grain": function (event, instance) {
```

Comment:
Wait, never mind. I was running a different version of the code than I thought I was.